ZhaoYuLin/Dog.java

```diff
@@ -6,19 +6,50 @@
     int age;
     boolean rabies;
 
-    public Dog(String name, String type, int age) {
+    public Dog(String name,String type,int age) {
         this.name = name;
         breed = type;
         this.age = age;
         rabies = false;
     }
-   
+
     public void bark() {
         System.out.println("bork");
     }
-    
+
     public static void main(String[] args) {
         Dog foo = new Dog("foo", "golden retriever", 5);
         foo.bark();
+        Dog Fred = new Dog();
```

Comment:
Object names are usually in camel case, meaning the first word is all lower case and then the following words have their first letter capitalized. 
Ex: dog, bigDog, veryBigDog, etc.

Fix the object name here.